codice/ddf
# DDF-3944 - Refactors location-old component to provide extensibility

#### Who is reviewing it? 

@mackncheesiest 
@adimka 

#### Select relevant component teams: 

@codice/ui

#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@bdeining

#### How should this be tested?

- `mvn clean install` ui directory
- ensure geo search still works as before
  - it might be best to have the previous version deployed to ddf and the PR changes loaded up in the dev server to compare/contrast the changes

#### What are the relevant tickets?

[DDF-3944](https://codice.atlassian.net/browse/DDF-3944)

#### Screenshots (if appropriate)

#### Checklist:


#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
